app/views/admin/edition_change_notes/edit.html.erb

```diff
@@ -20,6 +20,16 @@
         <%= render "govuk_publishing_components/components/button", {
           text: "Update"
         } %>
+      </div>
+    <% end %>
+
+    <%= form_tag [:admin, :edition_change_note], id: @edition_to_change, method: :delete do %>
```

Comment:
Just a nit, but i'm wondering if should live as an action here instead?
https://github.com/alphagov/whitehall/pull/7131/commits/ca725a3b36f9dd5c0346045585f352f223f8ecf5#diff-b44ff3403d3f2cddd475847e71d41b5181f2762af4ecfe920c2d460a6c6133d7R27-R31

Wdyt?

Reply:
Also, it's not a big deal, but should this have a confirm delete page?

Reply:
Yes, I think that's a good idea.  I'll add that in.